src/user/screens/profile.screen.js

```diff
@@ -84,10 +80,11 @@ class Profile extends Component {
       isPendingUser,
       isPendingOrgs,
       isPendingCheckFollowing,
-      navigation
+      navigation,
```

Comment:
I think maybe we can set up eslint & check lint rule before commit in next step to keep the coding style quality 😄